### ISSUE TITLE
Indicate if the current URL is referenced in Obsidian

### ISSUE THREAD

**Issue by crmfyi** [obsidianmd/obsidian-clipper#112]
Before trying this official Obsidian web clipper, I used [Obsidian Web](https://chromewebstore.google.com/detail/obsidian-web/edoacekkjanmingkbkgjndndibhkegad) which has a useful feature in it that I wish to see added here. 

If the web page from which I'm invoking the web clipper has already been referenced in Obsidian, I'm given an indication of it in the extension. I can choose to ignore it and save another note, but if it has been referenced, I can click the link to be taken to it in Obsidian. 

This is helpful as I might not remember whether I've got a note foe the current web page already in Obsidian, so I would know if I want to simply augment the current note or create a new one. Please see the attached files to show how this works in the Obsidian Web extension. 

This is what I see when I first save a page to Obsidian.
![First time saving to Obsidian](https://github.com/user-attachments/assets/614ebbf6-6d61-44dd-90fc-a68adca70d5f)

This is what I see when a note referencing this page already exists in Obsidian. 
![Link to current page already exists in Obsidian](https://github.com/user-attachments/assets/b24cd20d-b67b-4d3b-8e8f-0ca45d4d9958)

**Comment by kepano** [comment]
For now there are no plans to create a two-way integration between the Obsidian app and the browser extension. Data will be pushed into Obsidian, but the extension cannot access data in Obsidian. This is intentional. The extension you're referencing runs a local server which is outside of Obsidian's core capabilities currently.

**Comment by fabioscarsi** [comment]
@kepano, I respect your choices and excellent work with Obsidian Web Clipper, but the two-way integration would have a significant value. I know because I am a first-time adopter of Readwise Reader, and there, the integration is extremely useful for many reasons (I know the infrastructure is entirely different). Anyway, in the long run, I think @crmfyi Made a good point. At least take note. I wish I could code to help you with this great tool...

**Comment by crmfyi** [comment]
Thank you for your response @kepano . I do acknowledge that the other extension does use the Local Rest API plugin, which I do recognize not all users would want to leverage. In future iterations, it would be nice to have the option of leveraging what is in Obsidian in order to help users. Thank you for considering it. Considering this is an official Obsidian integration, it would be nice to have a switch to allow access to the vault if the user chose to allow it. I recognize that not all would want to use it that way.

**Comment by writtenfool** [comment]
I would not want, at anytime, for a web app to have access to my local app. There is a huge difference between web app to web app interfaces and a web app to a local OS app.

@fabioscarsi - there is a setting in Obsidian Clipper for each template that would alliviate your issue with note name, check out the Behavior setting. You have options... 

FWIW - there are many former early adopters of Readwise in the Obsidian space; they left us behind with their integration plugin and have still not figured out how to do it.  This is why Obsidian Clipper is a huge step up. I love Reader, but I got really tired of maintaining meta data in two places.

**Comment by fabioscarsi** [comment]
@writtenfool  I agree with your last consideration, which focuses on data duplication in several repositories. But this is why I hope that sometimes Obsidian + Obsidian Web Clipper could completely substitute Readwise Reader for all its functionalities, including the ones made possible during web navigation through two-way integration.

**Comment by CarcajadaArtificial** [comment]
Is it that bad to ask for this feature? I just want to not end up with duplicate web clippings. That's all. At least store in the extension's cache a simple list of urls that you've clipped before and simply show a warning if a new clip is found in the list.

**Comment by kepano** [comment]
It's possible, in fact the URL is stored as a history item in Web Clipper's logs. However there isn't a reliable link between that URL and the note in Obsidian because that note could have been renamed, deleted, moved, etc. I am not sure if it really provides enough utility to show "this page has been clipped before" without being able top open that note reliably?
